packages/contentful/src/crn/queries/manuscript.queries.ts

```diff
@@ -8,6 +8,13 @@ export const manuscriptContentQueryFragment = gql`
       id
     }
     title
+    teamsCollection(limit: 10) {
```

Comment:
Would we be having multiple teams assigned to a manuscript?

Reply:
Yeah, we're going to have this field in the form in the future :)
![Screenshot 2024-05-20 at 09 34 32](https://github.com/yldio/asap-hub/assets/16595804/e1c0c9c3-934e-428a-9f1c-cb9f93ebe508)

Reply:
Ah yes, I remember the discussion on this. Thanks! The teamId in the model confused me but it makes sense to add that when it becomes relevant. Looks good!

Reply:
thanks 😄